quickstarts/maven/micronaut/src/test/java/org/eclipse/jkube/quickstart/micronaut/custom/SimpleClient.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2019 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at:
+ *
+ *     https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.jkube.quickstart.micronaut.custom;
+
+import io.micronaut.http.annotation.Get;
+import io.micronaut.http.client.annotation.Client;
+import reactor.core.publisher.Mono;
+// import reactor.core.publisher.FluxOnErrorResume;
```

Comment:
nit, could you please remove this commented out line?

Reply:
Done